index.js

```diff
@@ -9,4 +9,10 @@
 
 server.listen(port, () => {
   console.log(`Server running on http://localhost:${port}/`);
-});
+});
+
+// Error handling
+server.on('error', (err) => {
+  console.error('Server error:', err);
+});
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While it's good that you've added error handling, it would be better to also handle the case where the error is fatal and the server should stop. For example, if the port is already in use, the server won't be able to start, and it would be appropriate to exit the process with a failure status code.
```diff
server.on('error', (err) => {
  console.error('Server error:', err);
+   process.exit(1);
});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->